.github/workflows/clean-artifacts.yml

```diff
@@ -1,17 +1,18 @@
+# Pretty sure this isn't needed, see attached screen shot of repo settings
```

Comment:
Is this file doing something different than this repo setting? if not we can remove this file and just update the repo setting. 

<img width="470" alt="Screen Shot 2022-10-18 at 2 58 49 PM" src="https://user-images.githubusercontent.com/15232757/196521193-fe5a97e7-7880-4cc3-a1e9-6c2595405df4.png">

Reply:
@evan-forbes thought on removing the workflow and just using this repo setting?  Do you remember why this workflow was added?

Reply:
yeah, I think we can delete this

Reply:
It was likely copied over from the sdk or default from startport